setup.py

```diff
@@ -19,6 +29,7 @@ def read(filename):
     platforms='any',
     packages=find_packages(),
     install_requires=read('requirements.txt').splitlines(),
+    tests_require=get_test_requirements(),
```

Comment:
I would add the requirements in setup.py, both for install and tests.
As for tests, there could then also be an `extras_require`, e.g.:
```python
    extras_require={
        'testing': DEPS_TESTING,
        'dev': DEPS_TESTING + DEPS_QA + [
            'pdbpp',
            'pytest-pdb',
        ],
        'qa': DEPS_QA,
    },
```

This allows to `pip install -e '.[testing]'` then.
`pip install -e .` uses the existing requirements (which should be moved from requirements.txt) to setup.py.